### PR TITLE
[GSoD] Update Troubleshooting

### PR DESCRIPTION
**Pull request by PROTechThor**
I basically just moved stuff around. There's no new content. 

The "Lost internet access after a TemplateVM update" seemed to be outdated (since 2013). I wasn't sure whether to keep it or not. I added a line indicating it may not be relevant anymore.

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
`dnf` update questions are definitely very common. Even if this is mostly moving stuff around, perhaps this is a good opportunity to have Marek review the content to see if any substantive improvements can be made.

File: user/troubleshooting/update-troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Run `systemctl enable NetworkManager-dispatcher.service` in the TemplateVM upon which your NetVM is based. 
+You may have to reboot afterward for the change to take effect. 
+(Note: This is an upstream problem. See [this Redhat ticket](https://bugzilla.redhat.com/show_bug.cgi?id=974811)). 
+For details, see the qubes-users mailing list threads [here](https://groups.google.com/d/topic/qubes-users/xPLGsAJiDW4/discussion) and [here](https://groups.google.com/d/topic/qubes-users/uN9G8hjKrGI/discussion).)
+
+## Windows update is stuck ##
+
+This has nothing to do with Qubes. 
+[It's a longstanding Windows bug.](https://superuser.com/questions/951960/windows-7-sp1-windows-update-stuck-checking-for-updates)
```
Comment (marmarek) [inline user/troubleshooting/update-troubleshooting.md:38]: I'd note also the link contains solutions for the issue.

File: user/troubleshooting/update-troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Dom0 and/or TemplateVM update stalls when updating via the GUI tool
+
+This can usually be fixed by updating via the command line.
+
+In dom0, open a terminal and run `sudo qubes-dom0-update`.
+
+In your TemplateVMs, open a terminal and run `sudo dnf upgrade`.
```
Comment (marmarek) [inline user/troubleshooting/update-troubleshooting.md:46]: This command applies to Fedora template only. For Debian it is `apt-get update && apt-get dist-upgrade`. You may also link to https://www.qubes-os.org/doc/software-update-domu/#updating-software-in-templatevms for details.

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by PROTechThor** [comment]
@marmarek All good now?

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Approved as of 3303fab5e76d2e4efbae40039fe2a7c7673d2a61
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAl+9xF8ACgkQ8yiUvpaE
k4ogtQ/7B8/IKgBc1EXMigPvC7Lsp0FvH6kellnCOuuYR1amA86LKXTlkwwZZyrd
/7PYpHygK6aQ6Txex8UqN2gyapdSIcH/OFHKVIIRoz9OufH+EbHKz0MuTf15lVVS
ZznFWtZQ7uWO7n2nhyzairm2tXakZuBSE/xMljLxAWWhrLn5Mv+3NForoKXw+V5J
3dIdMXwj6flrlfNLeHH4MdNC+REX7euiZYM1mTVzU9yTW5tN9ByQjkbP3NYezkbe
JosKcGcoby+jHT7tJmPsmRiDNSAfe+rmIv5IjQzthRoTT+12dAZlduCV8ZCInl5u
zaZ3g06/r71iCW+YcPxCcP4/gPM0w+NfDsLG2hcGPDDWcRBtVChds1jlmpZkDdbi
46L/l6+x3DptGr70a+g1uSdzdDnMysyW5K3nCKv9IR2A4R+phsJOYr6mkmrEi1IJ
k2tFVt2GIfWV5Vfy8vuy1Ncgy6osnCCX7Vjm53hKuH5vIIUXE5n/iFMjHB6dExAi
heAVR5MZpPwnnyjW3k56BZPbGGd/4gP9ZGGoxodbv6BWOI2NvOK1Naf7CkDIhme7
3+RyaAyKXsdy/tAovLcvLAI612UNOkSZQSc98O8iVSZ9zsxlrQJNFQXIv3+NTCAF
gwKUShMTNUM32gx37+vZzzttfDRXH6lYpBYM4PXDHn23QmbOLuQ=
=Pp+J
-----END PGP SIGNATURE-----
```

**Pull request merged**